app/models/account.rb

```diff
@@ -68,7 +68,7 @@ class Account < ApplicationRecord
     [:files, { action: :delete, class_name: 'CMS::File' }],
     [:builtin_pages, { action: :delete, class_name: 'CMS::BuiltinPage' }],
     [:provided_groups, { action: :delete, class_name: 'CMS::Group' }]
-  ].freeze
+  ]
```

Comment:
Why removing the `freeze`?

Reply:
because we modify in the audited method, cannot modify frozen

Reply:
Actually we may better just append to the existing empty array here too.